libs/blocks/ost/ost.js

```diff
@@ -6,7 +6,7 @@ const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
 const OST_VERSION = '1.11.0';
 const OST_BASE = `https://www.stage.adobe.com/special/tacocat/ost/lib/${OST_VERSION}`;
-const OST_SCRIPT_URL = `${OST_BASE}/index.js`;
+const OST_SCRIPT_URL = 'http://localhost:3000/libs/blocks/ost/index.js';
```

Comment:
This can't be right and is probably a left-over from testing.

Reply:
thanks